packages/synapse-react-client/src/lib/containers/auth/TwoFactorEnrollmentForm.tsx

```diff
@@ -61,8 +61,8 @@ const SectionInnerGrid: StyledComponent<BoxProps> = styled(
   columnGap: theme.spacing(5),
 }))
 
-// TODO https://sagebionetworks.jira.com/browse/DOCS-115
-export const TWO_FACTOR_DOCS_LINK = ''
+export const TWO_FACTOR_DOCS_LINK =
+  'https://help.synapse.org/docs/Managing-Your-Account.2055405596.html#Adding-Two-Factor-Authentication-(2FA)-to-your-account'
```

Comment:
The link will still work but the link to the section won't work until the [new doc](https://sagebionetworks.jira.com/wiki/spaces/DRAFT/pages/1972404729/Managing+Your+Account#Adding-Two-Factor-Authentication-(2FA)-to-your-account) is published